res/values-el/strings.xml

```diff
@@ -1296,21 +1296,21 @@ Commands:
     <string name="XP_5x">ΠΕ 5x</string>
     <string name="India">Ινδία</string>
     <string name="Tutorial">Διδακτικό μάθημα</string>
-    <string name="TYCOON">TYCOON</string>
+    <string name="TYCOON">ΜΕΓΙΣΤΑΝΑΣ</string>
     <string name="CLAN_BACKGROUND_COLOR">Χρώμα φόντου Κλαν</string>
     <string name="activate">Ενεργοποίηση</string>
     <string name="clan_id">Κλαν ID</string>
     <string name="language_change_warning">Η αλλαγή γλώσσας θα κάνει την εφαρμογή να επανεκκινηθεί.</string>
     <string name="xmas_gift_boost_message">Παραλάβατε ένα δώρο ενίσχυση 3x πλάσμα για 1 μέρα!</string>
     <string name="no_response">Δεν υπάρχει σύνδεση στον σέρβερ. Παρακαλώ ελέγξτε την σύνδεση σας.</string>
     <string name="RAMADAN">ΡΑΜΑΖΑΝΙ</string>
-    <string name="SUDDEN_ATTACK">SUDDEN ATTACK</string>
-    <string name="_2x_skin_resolution">2x Skin Resolution</string>
+    <string name="SUDDEN_ATTACK">ΞΑΦΝΙΚΗ ΕΠΙΘΕΣΗ</string>
+    <string name="_2x_skin_resolution">2x ΣΚΙΝ Ψήφισμα</string>
```

Comment:
In this case, "ΣΚΙΝ" shouldn't be 100% capital, only the first letter (Σ).

Reply:
Done I edited it
ID: 5906366

Reply:
Thank you ^^

Reply:
@7moxx 
Still there's wrong change. I said only the Sigma letter should be capital, and you left it small letter.

Reply:
> @7moxx Still there's wrong change. I said only the Sigma letter should be capital, and you left it small letter.

Because at the left (English translation) was skin small letters.  Any way I will made it as u said

Reply:
Thank you ^^
In this case, some English string values appear really as small letter.

**Tip:** You can change game to English to view better the texts, so you can translate it at the same way in Greek, or you can check `res/values`. ^^